pom.xml

```diff
@@ -107,6 +107,33 @@
           </includes>
         </configuration>
       </plugin>
+      <plugin>
+        <artifactId>maven-antrun-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>sample-data</id>
+            <phase>compile</phase>
+            <configuration>
+<target>
+   <mkdir dir="target/sample-data"/>
+   <defaultexcludes add="**/.DS_Store"/>
+   <zip destfile="target/sample-data/Dynamic Radar Coverage.mef"
+     basedir="src/main/plugin/iso19139.ca.HNAP/sample-data/Dynamic Radar Coverage"/>
+   <zip destfile="target/sample-data/Harbour Porpoise Presence.mef"
+     basedir="src/main/plugin/iso19139.ca.HNAP/sample-data/Harbour Porpoise Presence"/>
+   <zip destfile="target/sample-data/North American Radar.mef"
+     basedir="src/main/plugin/iso19139.ca.HNAP/sample-data/North American Radar"/>
+   <zip destfile="target/sample-data/Ocean Salmon Program.mef"
+     basedir="src/main/plugin/iso19139.ca.HNAP/sample-data/Ocean Salmon Program"/>
+   <zip destfile="target/sample-data/Potential Haul Out Sites.mef"
+     basedir="src/main/plugin/iso19139.ca.HNAP/sample-data/Potential Haul Out Sites"
+     includes="**"/>
+</target>
```

Comment:
This is responsible for creating mef files

Reply:
This is great :) - I always found it to be a pain to be committing zip files - nice to see that we can now version control individual files without having to commit a zip file.

Reply:
I should probably look at add-schema.sh a bit as it will only work when using the default unzipping the maven plugin artifact approach.

Still if you like this we could do it for other schema plugins also on core-geonetwork.